src/repositories/user.repository.ts

```diff
@@ -13,6 +13,12 @@ export const getUsers = async (): Promise<User[]> => {
     return users;
 };
 
+export const addUser = async (name: string, username: string): Promise<void> => {
+    const client = new Firestore();
+
+    await client.collection("users").add({ name, username });
```

Comment:
Does username need to be unique? Seems like it should be.

Reply:
@cstst fixed this!